src/apps/backend/modules/application/application-server.ts

```diff
@@ -1,11 +1,9 @@
+import { HttpStatusCodes } from 'backend/modules/application';
+import { ApplicationError } from 'backend/modules/application/application-error';
```

Comment:
Expose via index.ts in `application`, so other modules do not have to look beyond `backend/modules/application`